Makefile

```diff
@@ -40,7 +40,9 @@ docker.test: docker.images
 war: ${WAR}
 war.publish: ${WAR}
 	ssh ${SSH_OPTS} ${PKGSERVER} mkdir -p "'${WARDIR}/${VERSION}/'"
-	rsync -avz -e "ssh ${SSH_OPTS}" "${WAR}" "${PKGSERVER}:${WARDIR}/${VERSION}/${ARTIFACTNAME}.war"
+	sha256sum ${WAR} > ${WAR_SHASUM}
+	cat ${WAR_SHASUM}
+	rsync -avz -e "ssh ${SSH_OPTS}" "${WAR}" "${WAR_SHASUM}" "${PKGSERVER}:${WARDIR}/${VERSION}/${ARTIFACTNAME}.war"
```

Comment:
This can't possibly have worked, right? Two origin files, and a destination file name?